install/kubernetes/cilium/values.yaml

```diff
@@ -759,7 +759,7 @@ eni:
   # -- Enable Elastic Network Interface (ENI) integration.
   enabled: false
   # -- Update ENI Adapter limits from the EC2 API
-  updateEC2AdapterLimitViaAPI: false
+  updateEC2AdapterLimitViaAPI: true
```

Comment:
Just like CI reports, this change must go to `install/kubernetes/cilium/values.yaml.tmpl` in the first place.

(Then `make -C install/kubernetes` and `make -C Documentation update-helm-values`, but I think you've covered all the changes it should bring already?)